trade-finance-manager-ui/component-tests/case/facility/_macros/facility_details.component-test.js

```diff
@@ -189,106 +189,106 @@ describe(component, () => {
       wrapper.expectText('[data-cy="facility-tenor"]').toRead(params.facility.dates.tenor);
     });
 
-    describe('when facility end date feature flag is not enabled', () => {
+    describe('when shouldDisplayFacilityEndDate is false', () => {
       beforeEach(() => {
-        wrapper = render({ ...params, facility: facilityWithFacilityEndDate, isTfmFacilityEndDateFeatureFlagEnabled: false });
+        wrapper = render({ ...params, facility: facilityWithFacilityEndDate, shouldDisplayFacilityEndDate: false });
       });
       it('should not render isUsingFacilityEndDate', () => {
-        wrapper.expectText('[data-cy="is-using-facility-end-date"]').notToExist();
+        wrapper.expectText('[data-cy="facility-is-using-facility-end-date"]').notToExist();
       });
 
       it('should not render facilityEndDate', () => {
-        wrapper.expectText('[data-cy="facility-end-date"]').notToExist();
+        wrapper.expectText('[data-cy="facility-facility-end-date"]').notToExist();
```

Comment:
can the extra facility be removed from the data-cy

Reply:
I'm keeping convention with the existing `data-cy` namings on the `facility` page which is to have the word `facility` preceding the feature, so in this case it's the `facility-end-date` on the `facility` page and so has the name `facility-facility-end-date`

Reply:
I've changed this to `facility-end-date` as per discussion on another thread